tools/doc/addon-verify.mjs

```diff
@@ -1,23 +1,21 @@
-'use strict';
-
 // doc/api/addons.md has a bunch of code.  Extract it for verification
 // that the C++ code compiles and the js code runs.
 // Add .gyp files which will be used to compile the C++ code.
 // Modify the require paths in the js code to pull from the build tree.
 // Triggered from the build-addons target in the Makefile and vcbuild.bat.
 
-const { mkdir, writeFile } = require('fs');
-const { resolve } = require('path');
-const vfile = require('to-vfile');
-const unified = require('unified');
-const remarkParse = require('remark-parse');
-const gfm = require('remark-gfm');
+import { mkdir, writeFile } from 'fs/promises';
+
+import gfm from 'remark-gfm';
+import remarkParse from 'remark-parse';
+import { toVFile } from 'to-vfile';
+import unified from 'unified';
 
-const rootDir = resolve(__dirname, '..', '..');
-const doc = resolve(rootDir, 'doc', 'api', 'addons.md');
-const verifyDir = resolve(rootDir, 'test', 'addons');
+const rootDir = new URL('../../', import.meta.url);
+const doc = new URL('./doc/api/addons.md', rootDir);
+const verifyDir = new URL('./test/addons/', rootDir);
 
-const file = vfile.readSync(doc, 'utf8');
+const file = toVFile.readSync(doc, 'utf8');
```

Comment:
vfile doesn't support URL objects, I should have tested more carefully, it's a bit weird `to-vfile` doesn't throw an error.
```suggestion
const file = toVFile.readSync(fileURLToPath(doc), 'utf8');
```